core/modules/system/system.install

```diff
@@ -3360,6 +3363,13 @@ function system_update_1086() {
   }
 }
 
+/**
+ * Set the message length for the database log.
+ */
+function system_update_1087() {
+  config_set('system.core', 'log_max_length', 56);
```

Comment:
We try to prevent any loss of settings in the event this setting was pulled in via a config sync. Putting a safety check in here would be good:

```suggestion
  if (config_get('system.core', 'log_max_length') === NULL) {
    config_set('system.core', 'log_max_length', 56);
  }
```